### DISCUSSION TITLE
See rotation angle when rotating?

### DISCUSSION THREAD

**Discussion by Hipshot-zz** [Garux/netradiant-custom#183] [Q&A]
Is it possible to see the angle you rotate when you rotate a brush? Or is that displayed somewhere and I'm just not seeing it?

Like this: https://i.ibb.co/GWGsNR6/bild.png (TB)

I know this isn't displayed in old radiant, but it would be handy information to have.

**Comment by Garux** [comment]
This would be nice to have, probably even display params during any transform. Statusbar seems good place for numbers.

I think this is least practically useful for rotation compared to move or scale. For concrete angle you normally use `arbitrary rotation`, for 15` steps hold shift.
